app/stacks/uk-south/back-office/data.tf

```diff
@@ -10,3 +10,13 @@ data "azurerm_private_dns_zone" "app_service" {
 
   provider = azurerm.tooling
 }
+
+data "azurerm_key_vault_secret" "back_office_sql_server_password" {
```

Comment:
Swap these to output/input variables

Reply:
Fixed in https://github.com/Planning-Inspectorate/infrastructure-environments/pull/60/commits/8bf0fe49566d10f7b16de2f782ceea6dc0dafd83